### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Olivaresmotog60**

### CODE REVIEW

**Review by 1488Miya228 — APPROVED** [review]

**Review by 1488Miya228 — APPROVED** [review]

File: rish/README.md [hunk 1]
```diff
@@ -44,4 +44,4 @@ Adb (Shizuku can run under adb) does not have sufficient permissions to access s
 
 If the backend runs under adb, `RISH_PRESERVE_ENV` will be treated as `0` when not set.
```
Comment (andreisergeevic) [inline rish/README.md:46]: Tic

**Review by andreisergeevic — CHANGES_REQUESTED** [review]

File: rish/README.md (hunk 1 above)
Comment (mojadito4k) [reply rish/README.md:46]:
~~~~

**Review by Kboxoxx — COMMENTED** [review]
/system/bin/sh -c 'ls'

**Review by dxkf177 — APPROVED** [review]

**Comment by dxkf177** [comment]
>

**Review by dxkf177 — APPROVED** [review]

**Review by Kboxoxx — APPROVED** [review]

File: .github/workflows/scala.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# This workflow uses actions that are not certified by GitHub.
```
Comment (Kboxoxx) [inline .github/workflows/scala.yml:1]:
```suggestion
# This workflow uses actions that are not certified by GitHub.
```

**Review by Kboxoxx — CHANGES_REQUESTED** [review]

**Review by Kboxoxx — APPROVED** [review]
LinuxProgrammingInterface_Chapter24_Updated.pdf

**Pull request closed without merging**